pyproject.toml

```diff
@@ -97,49 +95,70 @@ style = "pep440"
 target-version = "py38"
 line-length = 127
 
+[tool.ruff.lint]
 select = [
-    "F",     # Pyflakes
-    "W",     # Pycodestyle (warnigns)
-    "E",     # Pycodestyle (errors)
-    "N",     # pep8-naming
-    "ANN",   # flake8-annotations
-    "B",     # flake8-bugbear
-    "FA",    # flake8-future-annotations
-    "TID",   # flake8-tidy-imports
-    "UP007", # flake8-new-union-types equivalent rule
-    "RUF",   # ruff's custom rules
+  "F",     # Pyflakes
+  "W",     # Pycodestyle (warnigns)
+  "E",     # Pycodestyle (errors)
+  "N",     # pep8-naming
+  "ANN",   # flake8-annotations
+  "B",     # flake8-bugbear
+  "FA",    # flake8-future-annotations
+  "TID",   # flake8-tidy-imports
+  "UP007", # flake8-new-union-types equivalent rule
+  "RUF",   # ruff's custom rules
 ]
 
 ignore = [
-    "ANN002", # Missing type annotation for *args
-    "ANN003", # Missing type annotation for **kwargs
-    "ANN101", # Missing type annotation for self in method
-    "ANN102", # Missing type annotation for cls in classmethod
-    "ANN204", # Missing return type annotation for special method
-    "B904",   # Exception raised within try-except should use raise ... from exc
-    "RUF012", # Mutable class attributes should be annotated with `typing.ClassVar`
+  "ANN002", # Missing type annotation for *args
+  "ANN003", # Missing type annotation for **kwargs
+  "ANN101", # Missing type annotation for self in method
+  "ANN102", # Missing type annotation for cls in classmethod
+  "ANN204", # Missing return type annotation for special method
+  "B904",   # Exception raised within try-except should use raise ... from exc
+  "RUF012", # Mutable class attributes should be annotated with `typing.ClassVar`
+
+  # Redundant rules with ruff-format:
+  "E111",   # Indentation of a non-multiple of 4 spaces
+  "E114",   # Comment with indentation  of a non-multiple of 4 spaces
+  "E117",   # Cheks for over-indented code
+  "D206",   # Checks for docstrings indented with tabs
+  "D300",   # Checks for docstring that use ''' instead of """
+  "Q000",   # Checks of inline strings that use wrong quotes (' instead of ")
+  "Q001",   # Multiline string that use wrong quotes (''' instead of """)
+  "Q002",   # Checks for docstrings that use wrong quotes (''' instead of """)
+  "Q003",   # Checks for avoidable escaped quotes ("\"" -> '"')
+  "COM812", # Missing trailing comma (in multi-line lists/tuples/...)
+  "COM819", # Prohibited trailing comma (in single-line lists/tuples/...)
+  "ISC001", # Single line implicit string concatenation ("hi" "hey" -> "hihey")
+  "ISC002", # Multi line implicit string concatenation
 ]
 
-[tool.ruff.extend-per-file-ignores]
+[tool.ruff.lint.per-file-ignores]
 "test_*.py" = [
-    "ANN", # flake8-annotations
+  "ANN", # flake8-annotations
 ]
 "docs/examples/code/**" = [
-    "FA",  # flake8-future-annotations
+  "FA", # flake8-future-annotations
```

Comment:
After looking a bit closer, I understood that ruff adds padding to comments, so between the longest string in TOML section and its comment would have one space. Like here:
```diff
 tomli = { version = "^2.0.1", markers = "python_version < '3.11'" }
-poetry-dynamic-versioning = "*" # actual version is in `release` group
+poetry-dynamic-versioning = "*"                                     # actual version is in `release` group
```
This is obviously a bug and should be reported to ruff, expected behavior should be to apply this padding only to comments that are going one by one, like here:
```diff
tomli = { version = "^2.0.1", markers = "python_version < '3.11'" }
-packaging = "24.0" # abc
+packaging = "24.0"             # abc
poetry-dynamic-versioning = "*" # actual version is in `release` group
```
Thought won't hold this PR because of this minor issue.

Reply:
Whoops, this was my bad, I run `taplo` LSP in my editor, which auto-formatted the TOML file like this. This formatting has nothing to do with ruff. Ruff only formats python files.

Reply:
TOML formatting changes were now reverted.